### DISCUSSION TITLE
Hey is there any way I could implement the new yolov10?

### DISCUSSION THREAD

**Discussion by TheWalKer-TW** [mikel-brostrom/yolo_tracking#1516] [Ideas]
I'm currently working in a project of mine using some parts of this project and the yolov8 model. I wanted to know if there is any way I could upgrade the yolo version to the new yolov10

**Comment by mikel-brostrom** [comment]
https://colab.research.google.com/drive/1-QV2TNfqaMsh14w5VxieEyanugVBG14V?usp=drive_link
